bin/mixer_codegen.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -e $ROOT/Gopkg.lock ]; then
   exit 1
```

Comment:
Should we rename this file to something like istio_codegen.sh since it's not only for mixer?

Reply:
maybe. but that's a bigger change (touches lots of files). I'd prefer to do that only as part of a larger cleanup once everything is working.